### ISSUE TITLE
AWS CLI example on wiki not working for me

### ISSUE THREAD

**Issue by m081072** [andrewgaul/s3proxy#109]
I follow the steps shown at https://github.com/andrewgaul/s3proxy/wiki/Client-compatibility-list and I get this:

```
:.aws $ aws configure
AWS Access Key ID [****************tity]: local-identity
AWS Secret Access Key [****************tial]: local-credential
Default region name [us-east-1]: None
Default output format [None]:
:.aws $ aws s3 ls --endpoint-url http://127.0.0.1:8080

A client error (InvalidArgument) occurred when calling the ListBuckets operation: Bad Request

:.aws $ aws --version
aws-cli/1.9.15 Python/2.7.6 Darwin/15.0.0 botocore/1.3.15
```

Am I doing something wrong, or is this a bug?

**Comment by m081072** [comment]
The above happened with 1.4.0

When I try with latest code, it works

**Comment by andrewgaul** [comment]
Duplicate of #104 and fixed in master via d8c73bc86d30e885e743eb7afcd41cf0b9e9497b.
